scripts/db/src/login.ts

```diff
@@ -14,8 +17,23 @@ type PersonResult = {
   last_name: string
   password: string
 }
+/** Forum user */
+type ForumUser = {
```

Comment:
Should this be in the `db-types` folder? Seems like you want to reuse this in the `getUser` script.

Reply:
I moved it when I moved Parfit PersonResult to be closer to the actual use-case, but actually I think in this case it's a feature of the database, not the query.